### DISCUSSION TITLE
How to connect a Tuya zigbee 3.0 Water/Gas Valve with zigbee2mqtt?

### DISCUSSION THREAD

**Discussion by Bcarfan** [Koenkk/zigbee2mqtt#9571] [Devices]
Hi. I'm trying to connect a zigbee 3.0 water/gas valve in zigbee2mqtt, but not happens. But in a generic zigbee bridge It connects ok. 
What can I do?
![20211108_200017](https://user-images.githubusercontent.com/93954172/140835770-b3a76dd0-4276-4b40-8b15-60f233eb74b8.jpg)

**Comment by uowis** [comment]
i would try this way
[manufacturerName](https://www.zigbee2mqtt.io/advanced/support-new-devices/02_support_new_tuya_devices.html#_2-adding-your-device)
 but i don"t have model id and no manufacturerName,...
 
![image](https://user-images.githubusercontent.com/1370666/230102637-f6e1d8d7-1770-4584-9ee9-a458f3f43be9.png)
